include/netuit/topology/DyadicTopologyFactory.hpp

```diff
@@ -50,7 +50,13 @@ struct DyadicTopologyFactory {
   netuit::Topology operator()(const size_t cardinality) const {
     return make_dyadic_topology(cardinality);
   }
+  netuit::Topology operator()(const emp::vector<size_t> cardinality) const {
+    emp_assert(cardinality.size() == 1);
+    return make_dyadic_topology(cardinality.front());
+  }
   static std::string GetName() { return "Dyadic Topology"; }
+  // todo: rename GetSlug, look into making static
```

Comment:
let's go ahead and rename it now, you can call slugify (inside emp/string utils) on GetName